Neos.EventSourcedNeosAdjustments/Classes/Testing/CustomizedInternalRequestEngine.php

```diff
@@ -0,0 +1,56 @@
+<?php
+namespace Neos\EventSourcedNeosAdjustments\Testing;
+
+/*
+ * This file is part of the Neos.ContentRepositoryMigration package.
+ *
+ * (c) Contributors of the Neos Project - www.neos.io
+ *
+ * This package is Open Source Software. For the full copyright and license
+ * information, please view the LICENSE file which was distributed with this
+ * source code.
+ */
+
+use Neos\Flow\Http\Client\InternalRequestEngine;
+use Neos\Flow\Session\SessionInterface;
+use Neos\Flow\Tests\FunctionalTestRequestHandler;
+use Neos\Flow\Http;
+use Neos\Flow\Http\Component\ComponentContext;
+
+class CustomizedInternalRequestEngine extends InternalRequestEngine
+{
+    public function sendRequest(Http\Request $httpRequest)
+    {
+        $requestHandler = $this->bootstrap->getActiveRequestHandler();
+        if (!$requestHandler instanceof FunctionalTestRequestHandler) {
+            throw new Http\Exception('The browser\'s internal request engine has only been designed for use within functional tests.', 1335523749);
+        }
+
+        // TODO: THE FOLLOWING LINE THIS IS THE ONLY CHANGE NEEDED!!!
+        //$this->securityContext->clearContext();
```

Comment:
commenting out this line is the only reason this class was copied. I would like it even more if we could emulate the Login properly; but I did not get it to work!